README.md

```diff
@@ -1,2 +1,190 @@
 # dpnt-video
-Datapoint - video
+
+Merge videos uploaded by user
+
+- [Hmmm](./container/images/hmmm/)
```

Comment:
There will be only one container. No need for multiple, language specific, containers.